src/components/tasks/TaskDetail.js

```diff
@@ -1228,9 +1228,9 @@ export class TaskDetail extends React.Component {
                                         Define frames to render. You can
                                         separate frame numbers
                                         <br />
-                                        with ;, eg. 1;4;7 will define frame 1, 4
+                                        with ; e.g. 1;4;7 will define frame 1, 4
                                         and 7. You can also define frames ranges
-                                        with -{" "}
+                                        with {" "}.
```

Comment:
`-` was there on purpose. Citing the documentation:

> You can also define frames ranges with -, eg. 1-4 will define frame 1, 2, 3 and 4

Reply:
Sorry then, I changed it because It was inconsistent with some other citings in TaskDetails.js.

Reply:
@mfranciszkiewicz What should I do about it? Could you suggest an edit?

Reply:
I believe @mdtanrikulu should share his thoughts on this.

Reply:
we can add double quotes to point the character. something like,  `with "-" character.` Also please add dot before `{" "}` Cuz it creates space between the following link and sentence.